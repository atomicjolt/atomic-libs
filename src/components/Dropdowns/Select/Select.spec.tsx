import React from "react";
import { describe, test, expect } from "vitest";
import { render } from "@testing-library/react";
import { Select } from ".";

test("matches snapshot", () => {
  const result = render(
    <Select value="none" label="Label" onChange={() => {}}>
      <option value="none">- Select an option -</option>
      <option value="value1">option 1</option>
      <option value="value2">option 2</option>
    </Select>
  );
  expect(result.asFragment()).toMatchSnapshot();
});
