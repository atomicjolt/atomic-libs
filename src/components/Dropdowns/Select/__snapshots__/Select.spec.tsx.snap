// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="aje-input is-medium"
  >
    <label
      class="aje-label"
      for="select-6009295216160224"
    >
      Label
    </label>
    <div
      class="aje-input__select"
    >
      <select
        aria-describedby=""
        id="select-6009295216160224"
      >
        <option
          value="none"
        >
          - Select an option -
        </option>
        <option
          value="value1"
        >
          Option 1
        </option>
        <option
          value="value2"
        >
          Option 2
        </option>
      </select>
    </div>
  </div>
</DocumentFragment>
`;
